src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java

```diff
@@ -69,36 +68,43 @@ public String launchInstructions(String trsID) {
     }
 
     @Override
-    public List<RowData> generateToolsTable(String initialPath, String contents, Map<String, Pair<String, GenericFileType>> indexedFiles) {
+    public List<RowData> generateToolsTable(
```

Comment:
Reformat for these parameters to be one per line is a bit odd, shouldn't be Dockstore style or Google style.

Reply:
it's from the mvn plugin. i didn't add it

Reply:
The mvn-maven-plugin? 
Weird, I assume they formatted their own code and it looks like 
https://github.com/coveooss/fmt-maven-plugin/blob/master/src/main/java/com/coveo/AbstractFMT.java#L224

Reply:
Not sure if they format their own code, but that example seems consistent.  If it's long, make it one per line, else make it one parameter per line.

Reply:
:shrug: